WordPress/Classes/ViewRelated/Media/Tenor/TenorAPI/TenorGIF.swift

```diff
@@ -0,0 +1,19 @@
+// Encapsulates Tenor GIF API object
+import Foundation
+
+struct TenorGIF: Decodable {
+    let id: String
+    let created: Date?
+    let title: String?
+
+    let media: [TenorGIFCollection]
+
+    let url: URL        // a short URL to view the post on tenor.com - we may not need this
```

Comment:
If we're not sure these are needed, what do you think about removing them and adding them later if they are actually needed?

Reply:
You're right. I have removed them ;-)